css/screen/large-screen.css

```diff
@@ -0,0 +1,193 @@
+.nav-date,
+.ul-section,
+.search-section,
+.trending-section,
+.weather {
+  display: block;
+}
+
+.weather-icon {
+  color: #fff;
```

Comment:
Can you confirm there's no bootstrap class for this?

Reply:
I will check it out, thanks